database/src/main/resources/db/migration/V10_0__add_coin_transfer.sql

```diff
@@ -0,0 +1,61 @@
+CREATE TABLE coin_transfer(
```

Comment:
since marker_transfer was already taken / in use, this seemed closest to existing convention of `coin_mint` and `coin_redeem_burn`